app.py

```diff
@@ -91,7 +93,7 @@ def login():
             if bcrypt.check_password_hash(db_user.password_hash, form.password.data):
                 print("valid password")
                 login_user(db_user)
-                return flask.redirect(flask.url_for("index"))
+                return flask.redirect(flask.url_for("bp.index"))
```

Comment:
Wasn't this working fine with "index" rather than "bp.index"?

Reply:
No this was not working with just index it just gave a user-visible error.

Reply:
What was the error? It worked fine for me before.

Reply:
This gave me an index.html missing jinja error.